website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -1,70 +1,81 @@
-#### Invocation
+## How to use
 
-To make the invocation more intuitive for developers, we've provided contextual components for the toggles and list-item items. For example, `<Hds::Dropdown::ListItem::Separator />` is yielded in a hash under the key `<XX.Separator />` when invoked:
+To make the invocation more intuitive for developers, we’ve provided contextual components for toggles and list items. For example, `<Hds::Dropdown::ListItem::Separator />` is yielded in a hash, using the key `<XX.Separator />` when invoked:
 
 ```handlebars
 <Hds::Dropdown as |dd|>
-  <dd.ToggleButton @text="..." />
-  <dd.Title @text="Lorem ipsum" />
-  <dd.Description @text="Lorem ipsum dolor sine qua non est." />
-  <dd.Interactive @href="..." @text="Add" />
+  <dd.ToggleButton @text="Menu" />
+  <dd.Title @text="Title Text" />
+  <dd.Description @text="Descriptive text goes here." />
+  <dd.Interactive @href="#" @text="Add" />
+  <dd.Interactive @href="#" @text="Add More" />
+  <dd.Interactive @href="#" @text="Add Another Thing Too" />
   <dd.Separator />
   <dd.Interactive @route="components" @icon="trash" @text="Delete" @color="critical" />
 </Hds::Dropdown>
 ```
 
-#### URLs and routes handling
+### ListItem::Interactive
 
-The `Interactive` list item renders the correct element based on the passing of an `@route`, `@href`, or the addition of a click event (i.e., `\{{on "click" this.myAction}}`).
+The `Interactive` list item renders the correct element based on the passing of an `@route`, `@href`, or the addition of a click event.
 
-_Notice: the `Interactive` list item component internally uses the generic `Hds::Interactive` component. For more details about how this low-level component works please refer to [its documentation page](/utilities/interactive/)._
+<!-- TODO This breaks the page...why?
+The `Interactive` list item renders the correct element based on the passing of an `@route`, `@href`, or the addition of a click event (i.e.,
+`{{on "click" this.myAction}}`). 
+-->
```

Comment:
Maybe you have to escape it? I saw this somewhere above:

\`\{{on "click" this.myAction}}`

Reply:
ahhh that worked. But so weird that it had to be that way- it should be already considered escaped by being a code sample. Oh well.